### PR TITLE
Tf api/fixes and cleanup

### PR DESCRIPTION
**Pull request by anamahalo**
## Labels

- [ ] inaccuracy
- [x] clarification
- [ ] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout
- [x] api - requires an update to the [changelog](https://www.terraform.io/docs/cloud/api/changelog.html)

API is marked but functionality is not new.
Please do not hesitate to reach out to me for questions!

### CODE REVIEW

File: content/source/docs/cloud/api/agent-tokens.md [hunk 1]
```diff
@@ -39,6 +39,15 @@ Status  | Response                                     | Reason
 [200][] | [JSON API document][] (`type: "authentication-tokens"`) | Success
 [404][] | [JSON API error object][]                    | Agent Pool not found, or user unauthorized to perform action
 
+### Query Parameters
+
+This endpoint supports pagination [with standard URL query parameters](./index.html#query-parameters); remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
+
```
Comment (laurapacilio) [inline content/source/docs/cloud/api/agent-tokens.md:45]:
```suggestion
This endpoint supports pagination [with standard URL query parameters](./index.html#query-parameters). Remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.

```

File: content/source/docs/cloud/api/agents.md [hunk 2]
```diff
@@ -45,9 +45,13 @@ Status  | Response                                     | Reason
 
 ### Query Parameters
 
+This endpoint supports pagination [with standard URL query parameters](./index.html#query-parameters); remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
```
Comment (laurapacilio) [inline content/source/docs/cloud/api/agents.md:48]:
```suggestion
This endpoint supports pagination [with standard URL query parameters](./index.html#query-parameters). Remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
```

File: content/source/docs/cloud/api/organization-memberships.html.md [hunk 3]
```diff
@@ -171,7 +171,7 @@ Parameter            | Description
 
 ### Query Parameters
 
-[These are standard URL query parameters](./index.html#query-parameters); remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
+This endpoint supports pagination [with standard URL query parameters](./index.html#query-parameters); remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
```
Comment (laurapacilio) [inline content/source/docs/cloud/api/organization-memberships.html.md:174]:
```suggestion
This endpoint supports pagination [with standard URL query parameters](./index.html#query-parameters). Remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
```

**Review by laurapacilio — APPROVED** [review]
Thank you! :)

File: content/source/docs/cloud/api/agent-tokens.md (hunk 1 above)
Comment (anamahalo) [reply content/source/docs/cloud/api/agent-tokens.md:45]: Thanks!  Given that the rest of the API docs are written this way going to keep as-is for consistency!

**Pull request merged**